Trakt/Api/TraktURIs.cs

```diff
@@ -88,12 +93,12 @@ public static class TraktUris
         /// <summary>
         /// The collected movies URI.
         /// </summary>
-        public const string CollectedMovies = BaseUrl + "/sync/collection/movies?extended=metadata";
+        public const string CollectedMovies = BaseUrl + "/sync/collection/movies";
 
         /// <summary>
         /// The collected series URI.
         /// </summary>
-        public const string CollectedShows = BaseUrl + "/sync/collection/shows?extended=metadata";
+        public const string CollectedShows = BaseUrl + "/sync/collection/shows";
```

Comment:
These changes seem unrelated to this PR?

Reply:
Indeed...